CONTRIBUTING.md

```diff
@@ -0,0 +1,92 @@
+## 📌How to Contribute📝:
+
+To contribute, look out for the issues raised by the maintainers as well as the other participants for contributions, and you can raise your issue also.
+
+### To start contributing, follow the below guidelines: 
+
+**1.**  Fork [this](https://github.com/SscSPs/color) repository.
+
+**2.**  Clone your forked copy of the project.
+
+```
+git clone https://github.com/<your_user_name>/SscSPs/color.git
+```
+
+**3.** Navigate to the project directory :file_folder: .
+
+```
+cd color
+```
+
+**4.** Add a reference(remote) to the original repository.
+
+```
+git remote add upstream https://github.com/SscSPs/color.git 
+```
+
+**5.** Check the remotes for this repository.
+
+```
+git remote -v
+```
+
+**6.** Always take a pull from the upstream repository to your master branch to keep it at par with the main project(updated repository).
+
+```
+git pull upstream main
+```
+
+**7.** Create a new branch.
+
+```
+git checkout -b <your_branch_name>
+```
+
+**8.** Perfom your desired changes to the code base.
+> 1. Add a screenshot of the feature you added or the change you made (This is important).
+
+**9.** Track your changes:heavy_check_mark: .
+
+```
+git add . 
+```
+
+**10.** Commit your changes .
+
+```
+git commit -m "Relevant message"
+```
+
+**11.** Push the committed changes in your feature branch to your remote repo.
+
+```
+git push -u origin <your_branch_name>
+```
+
+**12.** To create a pull request, click on `compare and pull requests`.
+
+**13.** Add appropriate title and description to your pull request explaining your changes and efforts done.
+
```

Comment:
Can you add a step here mentioning "In the PR, Add a screenshot of your change visible on UI(if possible)"
This should help reduce the time taken to verify the changes being proposed to be valid and not just spam

Reply:
Sure